screens/EventCreation/EventCreationScreen.tsx

```diff
@@ -69,7 +69,14 @@ const EventCreationScreen = ({ navigation, isAnnouncement }: EventCreationScreen
       showErrorToast("You must be logged in to create an event")
       return
     }
-    const eventId = await createEvent(title, description, date.toISOString(),{ x: 47.238458, y: 5.984155 }, location, "imageUrl", userId)
+    const eventId = await createEvent(
+      title, description, 
+      date.toISOString(), 
+      point || { x: 47.238458, y: 5.984155 }, 
```

Comment:
Do we fallback to that random place if the point can't be fetched?  i imagine this is due to typescript constraints but can't we should an `Alert` instead?

Reply:
Yepp you're right!